docs/guides/deploy.md

```diff
@@ -53,97 +58,95 @@ from concrete import fhe
 server = fhe.Server.load("server.zip")
 ```
 
-You will need to wait for requests from clients. The first likely request is for `ClientSpecs`.
+5. **Prepare for client requests**: The server needs to wait for the requests from clients. 
 
-Clients need `ClientSpecs` to generate keys and request computation. You can serialize `ClientSpecs`:
+6. **Serialize `ClientSpecs`**: The requests typically starts with `ClientSpecs` as clients need `ClientSpecs` to generate keys and request computation. 
 
 <!--pytest-codeblocks:skip-->
 ```python
 serialized_client_specs: str = server.client_specs.serialize()
 ```
 
-Then, you can send it to the clients requesting it.
+7. **Send serialized `ClientSpecs` to clients.**
 
-## Setting up clients
+### Setting up clients
 
-After getting the serialized `ClientSpecs` from a server, you can create the client object:
+8. **Create the client object**: After receiving the serialized `ClientSpecs` from a server, create the `Client` object.
 
 <!--pytest-codeblocks:skip-->
 ```python
 client_specs = fhe.ClientSpecs.deserialize(serialized_client_specs)
 client = fhe.Client(client_specs)
 ```
 
-## Generating keys (on the client)
+### Generating keys (client-side)
 
-Once you have the `Client` object, you can perform key generation:
+9. **Generate keys**: Once you have the `Client` object, perform key generation. This method generates encryption/decryption keys and evaluation keys. 
 
 <!--pytest-codeblocks:skip-->
 ```python
 client.keys.generate()
 ```
 
-This method generates encryption/decryption keys and evaluation keys.
 
-The server needs access to the evaluation keys that were just generated. You can serialize your evaluation keys as shown:
+10. **Serialize the evaluation keys**: The server needs access to the evaluation keys. You can serialize your evaluation keys as below.
 
 <!--pytest-codeblocks:skip-->
 ```python
 serialized_evaluation_keys: bytes = client.evaluation_keys.serialize()
 ```
-
-After serialization, send the evaluation keys to the server.
+11. **Send the evaluation keys to the server**.
 
 {% hint style="info" %}
-Serialized evaluation keys are very large, so you may want to cache them on the server instead of sending them with each request.
+Serialized evaluation keys are very large. Consider caching them on the server instead of sending them with each request.
```

Comment:
should we mention compressed keys here, @umut-sahin @BourgerieQuentin

Reply:
I think it'd be nice as an info box!

Reply:
Yes maybe we could add a sentence which redirect to https://docs.zama.ai/concrete/compilation/compression.

`Serialized evaluation keys are very large, even if there are [compressed](https://docs.zama.ai/concrete/compilation/compression) and can be reused several times, so you can consider caching them on the server ...`

Reply:
will be done in https://github.com/zama-ai/concrete-internal/issues/840, let's merge this PR to not make Yuxi wait